src/main/java/org/springframework/data/release/documentation/DocumentationCommands.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.data.release.documentation;
+
+import static org.springframework.data.release.model.Projects.*;
+
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.release.CliComponent;
+import org.springframework.data.release.TimedCommand;
+import org.springframework.data.release.cli.StaticResources;
+import org.springframework.data.release.model.TrainIteration;
+import org.springframework.shell.core.annotation.CliCommand;
+import org.springframework.shell.core.annotation.CliOption;
+
+/**
+ * @author Christoph Strobl
+ */
+@CliComponent
+@RequiredArgsConstructor
+public class DocumentationCommands extends TimedCommand {
+
+	private final @NonNull DocumentationOperations operations;
+
+	@CliCommand("docs check-links")
+	public void checkLinks(@CliOption(key = "", mandatory = true) TrainIteration iteration) {
```

Comment:
It would make sense to have a `--project` option to select a particular module. Useful when fixing links for a particular project.